devtools/__main__.py

```diff
@@ -1,15 +1,74 @@
-import click
+import subprocess
+import sys
+import os
+from pathlib import Path
+
+try:
+    import click
+    import rich
+    from .ctx import Context
+    from . import ci
+    from . import update_pyproject
+    from .subproject import Subproject
+    from .progress import progress
+except (ImportError, ModuleNotFoundError):
+    print("Installing robotpy dev requirements...")
+    subprocess.check_call(
```

Comment:
Definitely do not install the requirements automatically. Just detect the first import error and output the appropriate command for the user to copy/paste, but do not execute it.